src/styles.scss

```diff
@@ -4,31 +4,32 @@
 @import "./assets/scss/themes";
 @import "./assets/scss/mixins";
 @import "./assets/scss/bootstrapOverrides";
+@import "./assets/config/custom.css";
 
 @font-face {
   font-family: "Junicode";
-  src: url("/assets/fonts/Junicode.woff") format("woff");
+  src: url("assets/fonts/Junicode.woff") format("woff");
   font-weight: normal;
   font-style: normal;
 }
 
 @font-face {
   font-family: "Junicode";
-  src: url("/assets/fonts/Junicode-Bold.woff") format("woff");
+  src: url("assets/fonts/Junicode-Bold.woff") format("woff");
```

Comment:
Restore this modification. See comment on index.html

Reply:
See the mp